.changeset/gentle-actors-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+"@espressive/cascara": patch
+---
+
+chore(FDS-38):[Tests]: Improve unit test for input text
```

Comment:
```suggestion
chore(FDS-38):[DataText]: Improve unit tests
```

Reply:
@lordpixel @rociobaena remember that all of these show up on our change log. If I am a developer looking at changelogs for things that impact my work, I am not going to be looking through these titles for Tests. I am going to be looking through them by what part of the codebase it impacts. We should be putting brackets around the area we are making updates as the change being made instead of a generic "Tests" or "Cascara" in brackets.

Reply:
Ah!! Thanks for the detailed explanation here. When you mentioned what should be the label there, I thought it would be like a category, not a specific component.